modules/persistent-storage-csi-smb-cifs-provision-static.adoc

```diff
@@ -74,15 +74,15 @@ spec:
     - file_mode=0777
   csi:
     driver: smb.csi.k8s.io
-    volumeHandle: smb-server.default.svc.cluster.local/share/ <2>
+    volumeHandle: smb-server.default.svc.cluster.local/share/#subdir#app-data <2>
     volumeAttributes:
       source: //<hostname>/<shares> <3>
     nodeStageSecretRef:
       name: <secret_name_shares> <4>
       namespace: <namespace> <5>
 ----
 <1> The name of the PV.
-<2> `volumeHandle` format: {smb-server-address}.{sub-dir-name}.{share-name}. Ensure that this value is unique for every share in the cluster.
+<2> `volumeHandle` format: {smb-server-address}#{sub-dir-name}#{share-name}. Ensure that this value is unique for every share in the cluster, the `sub-dir-name` is optional, if use sub directory the sub directory must be exist otherwise mount would fail.
```

Comment:
Please reword as: "Ensure that this value is unique for every share in the cluster. The `sub-dir-name` is optional. If you are using a sub directory, the sub directory must be exist or the mount will fail.

Reply:
Addressed, thank you!